### PR TITLE
 ✨ Riverpod and autorouter added

### PR DESCRIPTION
**Pull request by Gokul2406**

### CODE REVIEW

**Review by Vortriz — APPROVED** [review]

File: lib/commons/widgets/loading_handlers.dart [hunk 1]
```diff
@@ -9,17 +9,23 @@ const LoadingDialogue({super.key, this.header, this.catLoading, this.footer});
...
     Widget build(BuildContext context) {
-
       return AlertDialog(
         title: const Text("Loading..."),
         content: Column(
           children: [
-            header ?? const Text("Please wait :D"),
-            catLoading ?? const CircularProgressIndicator(),
-            footer!,
+            testColumn()
```
Comment (pseudofractal) [inline lib/commons/widgets/loading_handlers.dart]: Need to remove testColumn()

File: lib/config/routes/router_config.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import 'package:auto_route/auto_route.dart';
+
+import '../../commons/widgets/firebase_init.dart';
+import '../../features/auth/interface/widgets/auth_checker.dart';
+import '../../features/homepage/interface/home_page.dart';
```
Comment (pseudofractal) [inline lib/config/routes/router_config.dart]: Need to add absolute paths from insight

File: lib/features/auth/interface/widgets/auth_checker.dart [hunk 3]
```diff
@@ -9,13 +10,13 @@ import 'package:insight/features/auth/providers/auth_provider.dart';
 
 import '../../../../commons/models/custom_errors.dart';
```
Comment (pseudofractal) [inline lib/features/auth/interface/widgets/auth_checker.dart]: Remove the relative file path here

**Review by pseudofractal — CHANGES_REQUESTED** [review]
Some changes to relative path and removal of test column.

**Review by pseudofractal — APPROVED** [review]
LGTM!

**Pull request merged**